core/embed/trezorhal/displays/st7789v.c

```diff
@@ -510,14 +280,22 @@ void display_init_seq(void) {
   HAL_Delay(120);
 
   // identify the controller we will communicate with
+#ifdef TREZOR_MODEL_T
   uint32_t id = display_identify();
   if (id == DISPLAY_ID_GC9307) {
-    send_init_seq_GC9307();
+    tf15411a_init_seq();
   } else if (id == DISPLAY_ID_ST7789V) {
-    send_init_seq_ST7789V(DISPLAY_ST7789V_INVERT_COLORS);
+    if (DISPLAY_ST7789V_INVERT_COLORS) {
+      lx154a2422_init_seq();
+    } else {
+      lx154a2411_init_seq();
```

Comment:
Samples of what?

Reply:
i actually thought this one was ok out of the box, but looking at it now side by side the `lx154a2422` colors now look more precise, but its quite close. nevertheless, there might be some room for improvement so i will query the producer one more time.

Reply:
adding the gamma settings recommended by the manufacturer here: https://github.com/trezor/trezor-firmware/commit/126ca70501b5cd3187b5634b5ebca303f78c53cf

it does make the image a bit lighter, but otherwise the difference in colors remains.